### PR TITLE
Dev andy

### PR DESCRIPTION
**Pull request by ahckim**
simple corruption manager
temporary jumping

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/FrozenMetalStudios/Soulless/pull/6" target="_blank">ReviewNinja

File: .gitignore [hunk 1]
```diff
@@ -71,3 +71,5 @@ Thumbs.db
 *.exe
 *.out
 *.app
+
+*.unity
```
Comment (DanApperloo) [inline .gitignore]: I don't think that we can do this, as the scene files are .unity files and we need those in git !fix

File: Project/Assets/Scripts/Utility/Logger.cs [hunk 2]
```diff
@@ -28,10 +28,10 @@ public enum eLogLevel
     };
 
 #if STANDALONE_MONOBEHAVIOUR
-    public class Logger : MonoBehaviour
+    public class ARK_Logger : MonoBehaviour
 #else
     [Serializable]
-    public class Logger
+    public class ARK_Logger
```
Comment (DanApperloo) [inline Project/Assets/Scripts/Utility/Logger.cs]: I think that c# Coding standards say that Classes should not contain _, so this should probably be ARKLogger !fix

File: Project/Assets/Scripts/Player/WalkableDetector.cs [hunk 3]
```diff
@@ -87,7 +87,7 @@ public bool CheckDownGround()
             return CheckImages(hits);
         }
 
-        public float check_height()
+        public float check_yPos()
```
Comment (DanApperloo) [inline Project/Assets/Scripts/Player/WalkableDetector.cs]: Function names should be CheckYPos !fix

Comment (DanApperloo) [inline Project/Assets/Scripts/Player/WalkableDetector.cs]: !fixed

File: Project/Assets/Scripts/Utility/Logger.cs (hunk 2 above)
Comment (DanApperloo) [inline Project/Assets/Scripts/Utility/Logger.cs]: !fixed

**Pull request merged**